hw/bsp/stm32f303disco/stm32f303disco.c

```diff
@@ -127,6 +129,9 @@ void board_init(void)
   SysTick_Config(SystemCoreClock / 1000);
   #endif
 
+  // Remap the USB interrupts
+  __HAL_REMAPINTERRUPT_USB_ENABLE();
```

Comment:
I pull out my f3 disco board to test with, I believe we need to enable the syscfg clk first for the remap to take effect.

```
__HAL_RCC_SYSCFG_CLK_ENABLE();
__HAL_REMAPINTERRUPT_USB_ENABLE();
``

Reply:
Whoops, I got distracted with work, so I forgot to test this in hardware. Fixed now.

Reply:
no problem at all, writing code is much harder than testing :)